samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -167,6 +222,21 @@ static void fill_raw_tx_pkt_hdr(struct raw_tx_pkt_header *raw_tx_pkt)
 	raw_tx_pkt->queue = CONFIG_RAW_TX_PACKET_SAMPLE_QUEUE_NUM;
 	/* The byte is reserved and used by the driver */
 	raw_tx_pkt->raw_tx_flag = 0;
+
+	if (validate(CONFIG_RAW_TX_PACKET_SAMPLE_RATE_FLAGS, 0, 4, "Rate Flags")) {
+		return -1;
+	}
+
+	if (validate_rate(CONFIG_RAW_TX_PACKET_SAMPLE_RATE_VALUE,
+			  CONFIG_RAW_TX_PACKET_SAMPLE_RATE_FLAGS)) {
+		return -1;
+	}
+
+	if (validate(CONFIG_RAW_TX_PACKET_SAMPLE_QUEUE_NUM, 0, 4, "Queue Number")) {
```

Comment:
Why is this validation needed here? Why can range be used in the Kconfig itself ?